PublicMapping/districtbuilder-website
# Update homepage sharing & lazy-loading

## Overview
Removes the "coming soon" badge on sharing (bc it's live, yay!) and implements lazy-loading (for homepage hero, contact page hero, featured image, CTA image at bottom of homepage, and article images). The latter will make the website more performant.

### Checklist

- [x] Deploy previews do not contain any content security policy errors

### Demo
<img width="1104" alt="Screen Shot 2020-11-30 at 5 58 38 PM" src="https://user-images.githubusercontent.com/5672295/100618886-d6811880-3335-11eb-9959-f4efbc6cd8d1.png">


### Notes
- I'm using https://github.com/verlok/vanilla-lazyload for lazy loading. To retain image proportions, I kept things simple and uploaded a tiny version of each image that is being loaded in this way.
- Also trying to use `<picture>` in more places for better performance on mobile. You can test that these are coming in by changing your screen width to <800px, ~900px, and >1000px and then right-clicking and looking at the "intrinsic" width/height:
<img width="979" alt="Screen Shot 2020-11-30 at 6 04 20 PM" src="https://user-images.githubusercontent.com/5672295/100619750-054bbe80-3337-11eb-80c2-2f797f150441.png">


## Testing Instructions
* `git pull`
* Go to the features section on the homepage. You shouldn't see the "Coming soon" badge on the 3rd item.
* If you scroll quickly down the page, you should see images fade in as you get to them. Right-click to see that they are given a ".lazy" class.

Closes #XXX
